src/clj/rems/auth/fake_login.clj

```diff
@@ -42,25 +51,38 @@
 
 (defn- user-selection [username]
   (let [url (url (login-url) {:username username})]
-    [:div.user.mb-3.mr-3.flex-fill.d-flex {:onclick (str "window.location.href='" url "';")}
-     [:a.btn.btn-primary.flex-fill {:href url} username]]))
+    [:div.user.m-3 {:onclick (str "window.location.href='" url "';")}
+     [:a.btn.btn-primary {:href url :style "width: 12rem; overflow: hidden; text-overflow: ellipsis; white-space: nowrap"}
```

Comment:
i noticed that we have a few places where the text truncation is used, and coincidentally bootstrap has a class that implements it: `text-truncate` https://getbootstrap.com/docs/4.6/utilities/text/#text-wrapping-and-overflow